Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM vault:0.9.1
+FROM vault:0.9.6
```

Comment:
We discussed tracking major versions, or even just latest, for stable third party docker images. Maybe this would be a good opportunity to apply this here?

In the case of nginx I combined this with an extra trigger on the TeamCity task that ran the build nightly, so that even if we're not pushing changes we'll stick pick up upstream changes regularly.

Reply:
Unfortunately vault does not do a 0.9 tag on hub: https://hub.docker.com/_/vault/

Reply:
Fair enough